_caca_alloc2d is an internal function and not exported to
the image viewer. It is basically a glorified call to malloc.

--- src/common-image.c.orig	2021-11-06 10:15:28.000000000 -0400
+++ src/common-image.c	2021-11-06 10:15:59.000000000 -0400
@@ -161,7 +161,7 @@
     uint32_t depth = (bpp + 7) / 8;
 
     /* Allocate the pixel buffer */
-    im->pixels = _caca_alloc2d(im->w, im->h, depth);
+    im->pixels = malloc(im->w * im->h * depth);
     if (!im->pixels)
     {
         caca_file_close(f);
